bfxapi/models/ticker.py

```diff
@@ -0,0 +1,72 @@
+"""
+Module used to describe all of the different data types
+"""
+
+class TickerModel:
+    """
+    Enum used to index the different values in a raw ticker array
+    """
+    BID = 0
+    BID_SIZE = 1
+    ASK = 2
+    ASK_SIZE = 3
+    DAILY_CHANGE = 4
+    DAILY_CHANGE_RELATIVE = 5
```

Comment:
I'd change 'RELATIVE' to 'PERC' here; it is a percentage (you note below in the constructor *100 for perc, but it's perc already)